### ISSUE TITLE
In Graphing mode, the keypad cannot be reached via the Tab key

### ISSUE THREAD

**Issue by joshkoon** [microsoft/calculator#948]
**Describe the bug**
In Graphing mode, users cannot use the Tab key to navigate to the keypad.

**Steps To Reproduce**

1. Go to Graphing mode
2. Switch to Equation mode if the window is too small to show all controls.
3. Use the Tab key to move focus through the app.
4. Observe that the keypad cannot be reached via the Tab key.

**Expected behavior**
Keypad should be included in the tab stops for the app. This also impacts navigation under Narrator.

**Screenshots**

**Device and Application Information**
 - OS Build: 10.0.19547.0
 - Architecture: x64
 - Application Version: 10.1912.4.0
 - Region: en-US
 - Dev Version Installed:

**Additional context**

**Requested Assignment**

I'm just reporting this problem.  I don't want to fix it.

**Comment by MicrosoftIssueBot** [comment]
This is your friendly Microsoft Issue Bot. I've seen this issue come in and have gone to tell a human about it.

**Comment by grochocki** [comment]
Thanks for the detailed bug report! This is _by design_ in Graphing Mode.

Unlike other calculator modes where the onscreen keyboard is locked to one display input/output area, Graphing Mode supports free-form input. While we still have the on-screen keyboard, it now behaves more like a SIP and is not intended to receive keyboard or screen reader focus. _All_ operations available in the SIP can be entered with standard keyboard input. The SIP is still available for simpler pointer input.

Free-from input in other modes is tracked by #620.
